src/lib/util.ts

```diff
@@ -382,6 +382,10 @@ export async function getNetworkName(id?: string): Promise<Networks> {
     id = await getNetworkId();
   }
 
+  if (typeof id !== "string") {
```

Comment:
Why this is necessary?

Reply:
because there are cases where the id is not a string . which comes from https://github.com/daostack/alchemy/blob/dev/src/lib/util.ts#L280 .  when connecting to WalletConnect